src/Library/demos/Column View/main.js

```diff
@@ -0,0 +1,101 @@
+import Gtk from "gi://Gtk";
+import Gio from "gi://Gio";
+
+const column_view = workbench.builder.get_object("column_view");
+
+function createFirstCol() {
+  const factory = new Gtk.SignalListItemFactory();
+  const columnViewColumn = new Gtk.ColumnViewColumn({
+    factory,
+  });
+  factory.connect("setup", (factory, listItem) => {
+    const label = new Gtk.Label({
+      height_request: 50,
+      margin_start: 12,
+      margin_end: 12,
+    });
+    listItem.set_child(label);
+  });
+  factory.connect("bind", (factory, listItem) => {
+    const labelWidget = listItem.get_child();
```

Comment:
Don't forget the project [code style](https://github.com/sonnyp/Workbench/wiki/Contribution-Style-Guide#general-principles-2) uses `camelCase` for function names, and`snake_case` for variable names.